java-symbolic-execution/java-symbolic-execution-plugin/src/test/java/org/sonar/java/se/checks/UnclosedResourcesCheckTest.java

```diff
@@ -33,6 +33,15 @@ void test() {
       .verifyIssues();
   }
 
+  @Test
+  void test_lombok() {
```

Comment:
Nitpick: I know it does not occur in the unit tests we have for rules, but I think it would be nice to rename this test with a more meaningful name. WDYT of  `does_not_raise_on_lombok_cleanup_annottated_variable`? It is kind of a mouthful but the contract would be clearer this way.

Reply:
Let's add another test without semantics

Reply:
Renamed with camelCase. Added a test `_noSemantic`.